daemon/src/to_sse_event.rs

```diff
@@ -318,7 +318,10 @@ impl ToSseEvent for CfdsWithAuxData {
                     margin: cfd.margin().expect("margin to be available"),
                     margin_counterparty: cfd.counterparty_margin().expect("margin to be available"),
                     details,
-                    expiry_timestamp: cfd.expiry_timestamp(),
+                    expiry_timestamp: match cfd.expiry_timestamp() {
```

Comment:
This is to avoid "not sending a timestamp" to the UI - in case we don't have a Dlc yet it do use the one from the order here for display in the UI.